app/views/two_factor_authentication/shared/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,18 @@
+<% lockout_time_in_words = decorated_user.lockout_time_remaining_in_words %>
+<% title t('titles.account_locked') %>
+
+<h1 class="h3 my0">
+  <%= t('titles.account_locked') %>
+</h1>
+<p>
+  <%= t('devise.two_factor_authentication.max_login_attempts_reached') %>
+</p>
+<p>
+  <%= t('devise.two_factor_authentication.please_try_again_html',
+  time_remaining: content_tag(:span, lockout_time_in_words, id: 'countdown')) %>
```

Comment:
The upside of this (vs having the HTML in the i18n string itself):

- feels more railsy 
- easier to read 

Downside:

- when just looking at the translation itself (`please_try_again_html`), it is not obvious that there is HTML used in the translation. Need to look at where it is called.

Because of this downside, I think we've mostly been including the HTML directly in I18n translation. At least that was the case the last time I checked.

What do you think?